cmd/sonobuoy/app/args/image.go

```diff
@@ -0,0 +1,37 @@
+package args
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// SonobuoyImage is the name of
+type SonobuoyImage string
+
+// AddSonobuoyImageFlag adds a sonobuoy-image flag to existing command
+func AddSonobuoyImageFlag(id *SonobuoyImage, cmd *cobra.Command) {
+	cmd.PersistentFlags().Var(
+		id, "sonobuoy-image",
+		"what container image to use for the sonobuoy worker and container",
```

Comment:
other commands have capitalized first words, don't really care if they are or not, but consistency is good